Tests/SwiftLayoutTests/LayoutDSLTest.swift

```diff
@@ -77,6 +77,19 @@ extension LayoutDSLTest {
         XCTAssertEqual(red.superview, root)
     }
     
+    func testSimpleWithViewConfiguration() {
+        deactivable = root {
+            UILabel().config { view in
+                view.text = "RED"
+                view.accessibilityIdentifier = "red"
+                return view
```

Comment:
Do I have to return it?
Is there any way to omit return?

Reply:
I've been trying a few things, and it's practically impossible. What do you think?

Reply:
@gmlwhdtjd baton passed

Reply:
> I've been trying a few things, and it's practically impossible. What do you think?

inout?

Reply:
> I've been trying a few things, and it's practically impossible. What do you think?

I found something!

Reply:
> > I've been trying a few things, and it's practically impossible. What do you think?
> 
> inout?

ok ok this should be already tested by you

Reply:
> > I've been trying a few things, and it's practically impossible. What do you think?
> 
> I found something!

oh!